src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -304,6 +305,8 @@ object WorkflowStatuses {
 
   def withName(name: String): WorkflowStatus = {
     name match {
+      case "Queued" => Queued
```

Comment:
I put this request into the queue doc - I want to settle on what we are going to call these states